src/Discovery/Utils/isManifestDataMissing.ts

```diff
@@ -0,0 +1,5 @@
+import { DiscoveryResource } from '../Discovery';
+
+const isManifestDataMissing = (resource: DiscoveryResource) => !resource.__manifest || resource.__manifest?.length === 0;
```

Comment:
let's use the `exportToWorkspace.manifestFieldName` from discovery config instead of hard coding `__manifest` here

Reply:
Updated as requested, the function now takes parameter ```manifestFieldName``` and the files DownloadZipButton.tsx, DownloadManifestButton.tsx and ActionButtons.tsx have been updated to pass this parameter when calling ```isManifestDataMissing```